src/pineko/check.py

```diff
@@ -168,40 +169,37 @@ def contains_ren(grid, max_as, max_al):
              max as order
     max_al : int
              max al order
+    sv_type : Scale
+        kind of scale_variation to be checked (either REN or FACT)
+
     Returns
     -------
-    bool
-        is ren scale-variation available for as
-    bool
-        is ren scale-variation available for al
+    : AvailableAtMax
+        result of the check
+    : int
+        effective max_as in the grid
```

Comment:
Since the information in the docstring is not telling much more than what the names are already telling, I'd propose you to move the types in the function type hints, and drop the `Parameters` and `Returns` sections.

Reply:
[d9b063c](https://github.com/NNPDF/pineko/pull/75/commits/d9b063c4be679386dcbe4fae246bf35756772716)